Formula/g/gemma.cpp.rb

```diff
@@ -1,23 +1,21 @@
 class GemmaCpp < Formula
   desc "Lightweight, standalone C++ inference engine for Google's Gemma models"
   homepage "https://github.com/google/gemma.cpp"
-  url "https://github.com/google/gemma.cpp/archive/refs/tags/v0.1.0.tar.gz"
-  sha256 "218280867d6f5256d3e3a9a34852e5752acd284538a8e7edaa6b382319e67c5c"
+  url "https://github.com/google/gemma.cpp/archive/refs/tags/v0.1.1.tar.gz"
+  sha256 "edc0529563114befcda414f0ae2ca0690b408e661ccfc3b61af360de53c00c4e"
   license "Apache-2.0"
 
   depends_on "cmake" => :build
   depends_on "sentencepiece"
 
   def install
-    system "cmake", "-B", "build", *std_cmake_args
-    cd "build" do
-      system "make", "gemma", "libgemma"
-      bin.install "gemma"
-      lib.install "libgemma.a"
-    end
+    system "cmake", "--preset", "make", *std_cmake_args
```

Comment:
Maybe like this?
```suggestion
    system "cmake", "--preset", "make", *std_cmake_args, "-DCMAKE_INSTALL_RPATH:STRING=#{rpath}"
```

Reply:
the ci still failed, maybe this not caused by rpath setting? 
<img width="1097" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/41824968/9edc06fd-6bc5-4f37-a0d4-e459f9dc99f0">
after installed `sentencepiece`, there should be a `libsentencepiece.0.dylib` file under `/usr/local/lib` dir. And `sentencepiece` has been declared as deps.
<img width="879" alt="Untitled" src="https://github.com/Homebrew/homebrew-core/assets/41824968/c1534ca8-3945-4965-b859-7f7fa8c5cfc6">

Reply:
I got the failed reason. CI runners installed `homebrew` under `/opt`. let me have another try.